backend-zs/routes/story.routes.js

```diff
@@ -1,11 +1,16 @@
 module.exports = storyRoutes
+var userController = require('../controllers/users.contoller')
 
 function storyRoutes() {
     var storiesController = require('../controllers/stories.controller')
     var router = require('express').Router()
 
     router.get('/', storiesController.getStories)
     router.get('/:id', storiesController.getStory)
+    router.post('/like/:id', storiesController.updateLike)
```

Comment:
One last comment, can we update the frontend to use this endpoint?

Reply:
I don't see the merged single story tab in this branch. I don't know why. I wanted to do this afterward and also add a fix in story list.

Reply:
Maybe, you didn't merge develop. However, it's ok to do both fixes together in a separate PR if you'd like.

Reply:
I will do them separately because they are both story and frontend related.